101-recovery-services-vm-workload-backup/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vaultName": {
+            "value": "SqlBackupDemoVault"
```

Comment:
use GEN-UNIQUE - or add this as a defaultValue to the template and just remove it

Reply:
Done.